governance/board-sync.py

```
@@ -0,0 +1,354 @@
+  updateProjectV2ItemFieldValue(input:$i){ projectV2Item{ id } } }"""
+
+FIELD_SPEC = [  # (字段名, 类型)——State 特殊（单选，选项=state 全集）
+    ("State", "SINGLE_SELECT"), ("Repo", "TEXT"), ("Assignee", "TEXT"),
+    ("卡号", "NUMBER"), ("停留天数", "NUMBER"), ("AC 进度", "TEXT"),
+]
+
+
+def ensure_project():
+    org = gql(Q_ORG, {"org": ORG})["organization"]
+    if org is None:
+        raise Infra(f"organization {ORG} 不可见（GOVERNANCE_TOKEN 权限？）")
```

Ruff (0.16.1)

[warning] String contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF001)